src/generate/index.ts

```diff
@@ -8,3 +8,32 @@ export { Api } from "./api";
 export { ApiGroup } from "./apiGroup";
 export { ApiCollection } from "./apiCollection";
 export { Template } from "./template";
+
+import {
+  getBaseUri,
+  getParameterDataType,
+  getPropertyDataType,
+  getRequestPayloadType,
+  getReturnPayloadType,
+  getProperties,
+  isAdditionalPropertiesAllowed,
+  isOptionalProperty,
+  isRequiredProperty,
+  isTypeDefinition
+} from "./handlebarsAmfHelpers";
+import { getValue } from "./utils";
```

Comment:
Why not just move getValue to handlebarsAmfHelpers or at least just export it from there? This now gives us another place to update when adding a helper.